Packs/MailListener/Integrations/MailListenerV2/MailListenerV2.py

```diff
@@ -623,7 +625,8 @@ def main():     # pragma: no cover
                                                       delete_processed=delete_processed, limit=limit,
                                                       save_file=save_file)
                 demisto.debug(f"{next_run=}")
-                demisto.setLastRun(next_run)
+                # if next_run is None, we will not update last_run
+                demisto.setLastRun(next_run) if next_run else None
```

Comment:
```suggestion
                if next_run:
                    demisto.setLastRun(next_run)
```

Not sure if this is automatic fix but its confusing and not worth the one-liner.

Reply:
Done